.gitignore

```diff
@@ -34,6 +34,24 @@
 *.out
 *.app
 
+## Compiler debris
```

Comment:
No big deal here, but for pull requests please try to address one aspect at a time. Changes to this file would be best done in a separate PR :)

Reply:
Moved to #280